### PR TITLE
fix: reccuring input width

### LINKED ISSUE #8220: [CAL-1468] Recurring Event input too wide
image

Can replicate here https://cal.com/hariom/recurring?duration=15

<sub>[CAL-1468](https://linear.app/calcom/issue/CAL-1468/recurring-event-input-too-wide)</sub>

### PR DESCRIPTION
**Pull request by joe-shajan**
## What does this PR do?

Fixes #8220

Before

image

After

image

**Environment**: Staging(main branch) / Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by joeshajan** [comment]
Right now this input box is duplicated in different places. Should we make it a component and reuse it?

**Comment by sean-brydon** [comment]
> Right now this input box is duplicated in different places. Should we make it a component and reuse it?

Hey bud we have - cal/packages/ui/components/form/inputs/Input.tsx. We could one create a reusable version for just NumberInput (Like we have for Email etc) or just use the Input defined here

**Review by sean-brydon — CHANGES_REQUESTED** [review]
Lets reuse our Input component :) 

Gimme a shout if you have any questions

**Comment by joeshajan** [comment]
> Lets reuse our Input component :)
> 
> Gimme a shout if you have any questions

Sure, I got you! I'll start working on it right away

**Comment by joeshajan** [comment]
Hey @sean-brydon, just wanted to let you know that I created a new NumberInput component and used it on the booking availability page. Can I use it in other places too? Waiting for your confirmation.

File: apps/web/components/booking/pages/AvailabilityPage.tsx [hunk 1]
```diff
@@ -226,16 +226,18 @@ const AvailabilityPage = ({ profile, eventType, ...restProps }: Props) => {
...
-                              type="number"
+
+                            <NumberInput
+                              defaultValue={eventType.recurringEvent.count}
                               min="1"
                               max={eventType.recurringEvent.count}
-                              className="w-15 border-default bg-default dark:border-empthasis h-7 rounded-sm text-sm font-medium [appearance:textfield] ltr:mr-2 rtl:ml-2"
-                              defaultValue={eventType.recurringEvent.count}
+                              isFullWidth={false}
+                              className="me-2 inline w-16"
```
Comment (joeshajan) [inline apps/web/components/booking/pages/AvailabilityPage.tsx:235]: should I add width here

Comment (sean-brydon) [reply apps/web/components/booking/pages/AvailabilityPage.tsx:235]: Yep :)

**Review by sean-brydon — APPROVED** [review]
Thank you for this :)

**Pull request merged**